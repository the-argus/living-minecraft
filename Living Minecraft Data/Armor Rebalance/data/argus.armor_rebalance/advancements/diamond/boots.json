{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:diamond_boots"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/diamond/boots"
  }
}
