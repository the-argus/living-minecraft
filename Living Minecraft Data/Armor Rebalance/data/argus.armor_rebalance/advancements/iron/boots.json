{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:iron_boots"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/iron/boots"
  }
}
