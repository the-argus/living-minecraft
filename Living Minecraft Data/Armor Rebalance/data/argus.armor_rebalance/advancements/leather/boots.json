{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:leather_boots"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/leather/boots"
  }
}
