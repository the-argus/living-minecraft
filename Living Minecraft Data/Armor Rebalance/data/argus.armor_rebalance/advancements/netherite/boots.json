{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "argus.armor_rebalance:netherite_boots_shapeless"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/netherite/boots"
  }
}
