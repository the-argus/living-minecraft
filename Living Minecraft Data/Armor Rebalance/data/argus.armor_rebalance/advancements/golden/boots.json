{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:golden_boots"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/golden/boots"
  }
}
